pkg/test/framework/runtime/components/apps/kube.go

```diff
@@ -272,10 +268,25 @@ func appSelector(serviceName string) string {
 }
 
 type kubeComponent struct {
-	scope       lifecycle.Scope
-	deployments []*deployment.Instance
-	apps        []components.App
-	env         *kube.Environment
+	scope lifecycle.Scope
+	// requiredApps is specified by the test.
+	requiredApps KubeAppsConfig
+	deployments  []*deployment.Instance
+	apps         []components.App
+	env          *kube.Environment
+}
+
+// KubeAppsConfig specifies Kubernetes app configuration.
```

Comment:
This is not a very useful comment, KubeAppsConfig == Kube Apps Config, you're just repeating the variable name.

Turn this into a useful comment or delete it.

Reply:
The lint will complain if no doc comment exists :)